README.md

```diff
@@ -77,7 +77,26 @@ To see default OpenDMARC configuration of this Docker image just run:
 docker run --rm instrumentisto/opendmarc cat /etc/opendmarc/opendmarc.conf
 ```
 
+#### Sending reports
 
+The Docker images come with msmtp MTA preinstalled which you can use to send reports when requested via the `ruf` tag inside a DMARC record.
```

Comment:
@schildbach I wonder whether some tests could be added for this?

At least, that `sendmail` is present in the image and is working.

Ideally, that `FailureReports true` works as expected.